### ISSUE TITLE
Suggestion: Config entry for default taxes

### ISSUE THREAD

**Issue by LlmDl** [TownyAdvanced/Towny#2094]
Originally reported on Google Code with ID 2071

```
Describe in detail what your suggestion is, *make sure you search ALL
issues,* do not submit something that has already been suggested:



I would like a funktion in the config file, where you can disable TOWNS from setting
there own plot and town tax. 
Also where you can disable the nations from setting tax.

Insteed of the towns and nations decide there own taxes i want this to be able to be
decided in the config file.

* Short version - I want to be able to decide all the taxes for the residents in the
config file.
```

Reported by `superola1986` on 2014-05-23 19:36:44
